jupyterlab_hub/typings/xterm/xterm.d.ts

```diff
@@ -0,0 +1,83 @@
+// Type definitions for term.js 0.0.7
```

Comment:
Typescript 2.0 introduces a new system for propagating typing information about 3rd party packages that would eliminate this, once these typings are stored in the central DefinitelyTyped repository. We need to submit the typings that we created for 3rd party libraries that aren't in that repository, but in the meantime, they may need to be copied.